rest/polygon_test.go

```diff
@@ -0,0 +1,121 @@
+package polygon_test
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	polygon "github.com/polygon-io/client-golang/rest"
+	"github.com/polygon-io/client-golang/rest/aggregates"
+	"github.com/polygon-io/client-golang/rest/client"
+)
+
+// todo: write some tests, just verifying that the client works for now
+
+func TestAggs(t *testing.T) {
+	c := polygon.New(client.HTTPBaseConfig{
+		URL:        "https://api.polygon.io",
+		Key:        os.Getenv("API_KEY"),
+		MaxRetries: 3,
+	})
+
+	res, err := c.Aggregates.Get(context.Background(), aggregates.GetParams{
+		Ticker:     "AAPL",
+		Multiplier: 1,
+		Resolution: "day",
```

Comment:
I think Resolution should be a type. There are a fixed size of options for this, and sometimes its easy to forget if its "day" or "days". Having it be a type allows the IDE to list what is available while you're using the library which is nice DX. I think we should do this for most enum values. ( Sort direction, etc ).

Reply:
totally agree on this, we can run through and add types for any enum style params